### DISCUSSION TITLE
Official PlayStation 3 Development Kit (Hardware) · RetroReversing

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#124] [Post Comments]
# Official PlayStation 3 Development Kit (Hardware) · RetroReversing

Learn about RetroReversing, the home of retro game reverse engineering enthusiasts. Get resources, tutorials, and tools for analyzing vintage technology and join the community today!

https://www.retroreversing.com/official-playStation3-devkit

**Comment by Ibram-RIMS** [comment]
How much is the ram in DECR-1400 Reference Tool it said double the ram of the non-dev ps3 but thus that mean double of 256mb or 512mb? If is double of 256mb that make sense at the time if not then it would mean double of 512mb and that doesn’t seem right for that time.

**Reply by Ibram-RIMS** [reply]
Or double of the ram of the other ver of the dev kit ps3?
